### PR TITLE
Add Credit Agricole to list of production users.

### PR DESCRIPTION
**Pull request by balchua**
Adding Credit Agricole CIB to the list of production users.

### CODE REVIEW

**Comment by porunov** [comment]
Hi @balchua ,
Thank you for adding a new production use-case user. Would you be able to share a little bit of information about the use case in Credit Agricole CIB? I couldn't find any public information about Credit Agricole CIB to use JanusGraph. 
If you are working in Credit Agricole CIB you could post a use case in the [GitHub discussions](https://github.com/JanusGraph/janusgraph/discussions/categories/show-and-tell), [users mailing list](https://lists.lfaidata.foundation/g/janusgraph-users/topics), or add a link to a blog post / conference video or slides.
If you are not working in Credit Agricole CIB but just know about the use case, it would be great if you could share where you found such information.
Usually to include someone into the production list we need an approval from either an author who is part of the entity or a representative of such entity.
Also, if you are interested to include a logo to https://janusgraph.org (Users section) we must get an approval from a representative of such an entity.

**Comment by balchua** [comment]
Hi @porunov
Yes i am working at Credit Agricole CIB.
We don't currently have public information on how we use JanusGraph with in our information system.
Do you have a link or sample in your Github Discussions forum where use cases are described?

Thanks again.

**Comment by porunov** [comment]
@balchua 
Here you can find several examples where users describe their JanusGraph usage in production:
https://lists.lfaidata.foundation/g/janusgraph-users/topic/79935292#1899
https://www.slideshare.net/RoopaTangirala/polyglot-persistence-netflix-cde-meetup-90955706#86
https://www.gdatasoftware.com/blog/2018/11/31203-malware-analysis-with-a-graph-database
https://www.youtube.com/watch?v=EtB1BPG00PE
https://www.scylladb.com/2020/05/14/zeotap-a-graph-of-twenty-billion-ids-built-on-scylla-and-janusgraph/
https://www.scylladb.com/2021/05/06/mapped-a-new-way-to-control-your-business-via-iiot/
https://www.scylladb.com/2021/03/11/qomplx-using-scylla-with-janusgraph-for-cybersecurity/
https://www.linkedin.com/feed/update/urn:li:activity:6693521144605089792

As for GitHub Discussions we didn't have any posts about production use cases there but that's mostly because we enabled GitHub Discussions only in December 2020 and also companies usually use other forms of posts (as seen above, those are mostly posts on different web sites). That said, you are very welcome to make the first post in GitHub Discussions under [Show and tell](https://github.com/JanusGraph/janusgraph/discussions/categories/show-and-tell) category.

**Comment by balchua** [comment]
Super!!! 👍

**Comment by li-boxuan** [comment]
@balchua To my best knowledge, financial services companies, including investment banks, typically have strict compliance rules about disclosing their brands/logos on third-party websites. It's great to see your company added to the production user list, but I would suggest confirming with your company's compliance team first.

**Comment by balchua** [comment]
@li-boxuan yeah im getting the use case approval from our compliance.  Sorry its taking a bit of time.  Work got in the way too.

**Comment by balchua** [comment]
Finally got approval from our compliance to post our use case into the discussion.

https://github.com/JanusGraph/janusgraph/discussions/2734

File: README.md [hunk 1]
```diff
@@ -122,6 +122,7 @@ CLAs and best practices for working with GitHub.
 The following users have deployed JanusGraph in production.
 
 * [CELUM](https://www.celum.com/) - [use case and system architecture](https://www.celum.com/en/graph-driven-and-reactive-architecture)
+* [Crédit Agricole CIB](https://www.ca-cib.com/)
```
Comment (porunov) [inline README.md:125]: Could you please also add a link to your use-case?
I.e. something like this:

[Crédit Agricole CIB](https://www.ca-cib.com/) - [use-case](https://github.com/JanusGraph/janusgraph/discussions/2734)

**Comment by balchua** [comment]
The EasyCLA seems to be down.

File: README.md (hunk 1 above)
Comment (balchua) [reply README.md]: done

**Comment by porunov** [comment]
@balchua could you please also squash your commits into one? 

@mbrukman do you know what to do when EasyCLA is down?

**Comment by balchua** [comment]
@porunov how do i squash these commits into one?

**Comment by balchua** [comment]
commit merged into 1

**Comment by porunov** [comment]
@balchua I didn't check but did you sign your commit?

I.e. you can do:
```
git commit --amend -s
git push -f
```

**Comment by balchua** [comment]
Yes i did.  Otherwise DCO check will not pass.

**Comment by porunov** [comment]
@balchua I opened the issue here: https://github.com/communitybridge/easycla/issues/3090

**Review by porunov — APPROVED** [review]
LGTM. Thank you @balchua !

**Review by li-boxuan — APPROVED** [review]
Great!

**Comment by porunov** [comment]
@balchua in case `Credit Agricole CIB` is OK to be added to https://janusgraph.org , let us know and we can place the logo on the main page.

**Comment by balchua** [comment]
@porunov thanks!  I will get back to you on this.  In case we are interested, what does it take to get it to janusgraph.org?
Thanks

**Comment by mbrukman** [comment]
@balchua — the process is very simple:

1. Please make sure your PR/Legal/etc. teams are OK with this.
2. Let us know what logo file you'd like us to use. Ideally, a scalable SVG would be best, but PNG will work as well.
 
That's it.

We'll want to check in the logo file into our website repo (if we use an external reference, it'll break every time the websites are reorganized, which is often).

**Comment by balchua** [comment]
Hi guys.  Finally got the approval from our comms team.  The logo is in `png`.
Should i checkout and submit a PR to the janusgraph.org repo?

**Comment by porunov** [comment]
That's great news! If you want, you can submit PR yourself (preferable) or just provide us with the logo.
If you wish to submit a PR you need to add 2 things into your PR.
1. Add markdown post file into the next directory (see how other companies submitted the PR): 
https://github.com/JanusGraph/janusgraph.org/tree/master/_posts
2. Add logo file here: https://github.com/JanusGraph/janusgraph.org/tree/master/img/logos

That's it. The logos will be picked up and restructured automatically. Notice that the name of the logo will be used in the markdown file. Also, notice that the name of the markdown file have a required date. Moreover, CI will fail in your PR but that's fine.

**Comment by balchua** [comment]
If you're fine im uploading the logo here.
Really appreciate the help.
CACIB

**Comment by porunov** [comment]
Thanks! Opened a PR here: https://github.com/JanusGraph/janusgraph.org/pull/85
Preview: https://porunov.github.io/janusgraph.org

**Pull request merged**